startup.d/20-syntax-check.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+python -m py_compile $(find . -name '*.py');
+if [ "$?" != "0" ]
+then
+  echo "There is syntax error. This may endanger chaos. Pauses updates."
+  killall bash
```

Comment:
@reddraggone9 , chaos.py is [a daemon](https://github.com/chaosbot/chaos/blob/ff46d4b06d5c3d97b5fbb754b9c01a1e985e33e0/chaos.py#L81). Demons use in principle use the version loaded into memory. Supervisord restart the demon if them crash. If no fail there is no restart, so version loaded into memory is used.  Until restart chaos.py use in-memory version.